_docs_integrate/11-connector-configuration.md

```diff
@@ -275,73 +278,13 @@ interface WebhooksModulePayload {
     messages: Message[];
     relationships: Relationship[];
 }
-
-interface Message {
-    id: string;
-    content: any;
-    createdBy: string;
-    createdByDevice: string;
-    recipients: Recipient[];
-    relationshipIds: string[];
-    createdAt: string;
-    attachments: string[];
-}
-
-interface Recipient {
-    address: string;
-}
-
-interface Relationship {
-    id: string;
-    template: RelationshipTemplate;
-    status: string;
-    peer: string;
-    changes: RelationshipChange[];
-    lastMessageSentAt?: string;
-    lastMessageReceivedAt?: string;
-}
-
-interface RelationshipTemplate {
-    id: string;
-    isOwn: boolean;
-    createdBy: string;
-    createdByDevice: string;
-    createdAt: string;
-    content: any;
-    expiresAt?: string;
-    maxNumberOfRelationships?: number;
-}
-
-interface RelationshipChange {
-    id: string;
-    request: {
-        createdBy: string;
-        createdByDevice: string;
-        createdAt: string;
-        content?: any;
-    };
-    status: "Pending" | "Rejected" | "Revoked" | "Accepted";
-    type: "Creation" | "Termination" | "TerminationCancellation";
-    response?: {
-        createdBy: string;
-        createdByDevice: string;
-        createdAt: string;
-        content?: any;
-    };
-}
 ```
 
-The payload of the webhook is the same as the response payload of the `/api/v1/Account/Sync` endpoint. Thus the type `ConnectorSyncResult` of the [TypeScript SDK](./connector-sdks#typescript-sdk) can be used for specifing the webhook's payload type.
+The payload of the webhook is the same as the response payload of the `/api/v1/Account/Sync` endpoint. Thus the type `ConnectorSyncResult` of the [TypeScript SDK](./connector-sdks#typescript-sdk) can be used for specifing the webhook's payload type. Klick the links for a
```

Comment:
> The payload of the webhook is the same as the response payload of the `/api/v1/Account/Sync` endpoint.

Hmm, das ist zumindest derzeit noch so. Aber auch nur durch Zufall. Wenn wir das in Zukunft mal ändern, müssten wir dran denken, diese Stelle hier abzuändern. Und ich weiß genau, dass ICH nicht dran denken würde. :) Also vielleicht lieber die Types direkt hier rein kopieren.

Reply:
rly? Das kam doch gar nicht über den PR rein ..

Reply:
und die types sind hier drinnen